### ISSUE TITLE
[FSR] post-9/11 GI Bill waiver resolution verification

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#18540]
## Issue Description
_If a Veteran submits an FSR for a waiver on a Post-9/11 GI Bill and it's granted, their remaining education benefits are reduced to resolve the debt. This may not be clear to our users, so we want to verify that they understand that their debt won't be completely waived if they choose this resolution._

---
## Tasks
- [x] _Update the debt resolution selection page in the UXPin prototype to require the Veteran to verify they understand, "This will reduce your remaining education benefits entitlement" before proceeding_
- [x] _Create a ticket for frontend dev team_

## Acceptance Criteria
- [x] _The UXPin prototype has been updated and linked to this ticket_
- [x] _A ticket for the frontend team has been created and linked to this ticket_

**Comment by rileyorr** [comment]
I updated the UXPin prototype to include a required checkbox confirming that the Veteran understands that [requesting a waiver on a Post-9/11 GI Bill debt](https://preview.uxpin.com/803dc24213cf723bfce6c5e3eaef5a2c3f3e344b#/pages/135678340/simulate/sitemap?mode=i) could result in their remaining education benefits being reduced. I also added a mockup for r[equesting a waiver on a Comp & Pen debt](https://preview.uxpin.com/803dc24213cf723bfce6c5e3eaef5a2c3f3e344b#/pages/135678339/simulate/sitemap?mode=i) indicating that checkbox isn't needed. I'm going to send this copy to the DMC team to make sure the statement is accurate.

**Comment by rileyorr** [comment]
I updated the design based on this morning's meeting with the DMC. We're now waiting on the DMC team to provide us the education benefits contact center's information after confirming we should direct them for questions about their remaining entitlement reducing

**Comment by rileyorr** [comment]
Updated the content with the education contact center's info and created [a ticket for the frontend team](https://github.com/department-of-veterans-affairs/va.gov-team/issues/18693). Peggy/Danielle is currently reviewing the contact, so I added a dependency on the frontend ticket

**Comment by rileyorr** [comment]
We're still waiting for content feedback, but I am going to move this ticket over and create a separate ticket to incorporate any content feedback so this work is documented for this sprint.
